src/components/Game/Result/Result.js

```diff
@@ -2,26 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
-const ResultView = ({ gameWon, correctWords }) => (
-    <div>
-        {
-            (gameWon)
-                ? 'gratulacje, wygrałeś'
-                : (
-                    `nie udało się, może następnym razem.
-                    poprawne słowa to:
-                    ${correctWords.map((word) => word.toUpperCase()).join(', ')}`
-                )
-        }
-    </div>
-);
+const ResultView = ({ words, answer }) => {
+    const isAnswerCorrect = () => words.includes(answer);
```

Comment:
later on, when it becomes separate Route, this computation can be done in `mapStateToProps`, it would be nice solution